modules/ipi-install-removing-the-provisioner-node.adoc

```diff
@@ -0,0 +1,13 @@
+// This is included in the following assemblies:
+//
+// installing/installing_bare_metal_ipi/ipi-install-post-installation-configuration.adoc
+
+[id="ipi-install-removing-the-provisioner-node_{context}"]
+= Removing the provisioner node
+
+Optionally, the provisioner node can be removed after installation.
+Make sure to back up the following files:
+
+. The `install-config.yaml` file in case the cluster must be redeployed
+. The generated `<kubeconfig>` and `<kubeadmin-passwd>` from the `</path/to/install/dir/auth/>` directory
+. The SSH key pair to SSH to a node
```

Comment:
I would include 'required' here for clarity and so read
The SSH key pair required to SSH to a node.

Reply:
Updated as suggested.